src/modules/application/containers/TradeWidgetLinks/index.tsx

```diff
@@ -40,9 +40,6 @@ export function TradeWidgetLinks() {
           to={parameterizeTradeRoute(tradeContext, Routes.LIMIT_ORDER)}
         >
           <Trans>Limit</Trans>
-          <styledEl.Badge>
-            <Trans>Beta</Trans>
-          </styledEl.Badge>
```

Comment:
should you move these a few lines down (and add it to TWAP)?

Even if its not the final design, it will make it so we don't forget adding it